ee_plugin.py

```diff
@@ -130,8 +130,16 @@ def updateLayers(self):
             ee_object = l.customProperty('ee-object')
             ee_object_vis = l.customProperty('ee-object-vis')
 
+            # check for backward-compatibility, older file formats (before 0.0.3) store ee-objects in ee-script property an no ee-object-vis is stored
+            # also, it seems that JSON representation of persistent object has been changed, making it difficult to read older EE JSON
+            if ee_object is None:
+                QgsMessageLog.logMessage('Map layer saved with older version of EE plugin is detected, backward-compatibility for versions before 0.0.3 is not supported due to changes in EE library, please re-create EE layer by re-running the Python script')
```

Comment:
the `QgsMessageLog.logMessage` send the message to the "Log messages" tabs, however normally the people use this plugin with the Python console and very surely that the user won't see this warning. How about printing this warning in the python console directly, like `print("\nWARNING:\n Map layer saved with older version..... \n")` ?

Reply:
or optionally we can use the message bar:  iface.messageBar().pushMessage("EE_plugin", "Map layer saved with older version.....", level=Qgis.Warning)

Reply:
Good point, changed